packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts

```diff
@@ -72,40 +74,53 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
   }
 
   const conditionalRetry = function* (id: string) {
-    const { RETRY_AMOUNT, SECONDS } = POLLING
-    const data = yield retry(RETRY_AMOUNT, SECONDS * 1000, transferAccountState, id)
-    return data
-  }
+    const MAX_TRIES = Array(6)
```

Comment:
why isn't this a number?

Reply:
For my `conditionalRetry` function I didn't want to deal with decrementing and checking if the value was at 0 so i just used an array + array.pop() + a while loop to create the effect I wanted.